yaml/thz504.yaml

```diff
@@ -5,3 +5,7 @@ esphome:
 
 packages:
   BASE: !include { file: wp_base.yaml}
+  ABLUFTTAUPUNKT:                 !include { file: thz504_temperature.yaml, vars: { property: "ABLUFTTAUPUNKT"   }} 
```

Comment:
If you want, we can merge this as a separate PR 

[minor] could you group the sensors together and separate them with a newline from the base?